lib/LedgerNodePeers.js

```diff
@@ -0,0 +1,145 @@
+/*!
+ * Ledger node operations management class.
+ *
+ * Copyright (c) 2017-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const bedrock = require('bedrock');
+const database = require('bedrock-mongodb');
+const {util: {BedrockError}} = bedrock;
+const {COLLLECTION_NAME} = require('./ledgerNodePeerCollection');
+
+/**
+ * The LedgerNodePeers class exposes peer information.
+ */
+module.exports = class LedgerNodePeers {
+  constructor({ledgerNode}) {
+    this.ledgerNode = ledgerNode;
+    this.ledgerNodeId = ledgerNode.id;
+  }
+
+  async add({peer}) {
+    const collection = database.collections[COLLLECTION_NAME];
+    const now = Date.now();
+    const peerRecord = {
+      meta: {
+        created: now,
+        ledgerNodeId: this.ledgerNodeId,
+        updated: now
+      },
+      peer,
+    };
+    let result;
+    try {
+      result = await collection.insertOne(peerRecord);
```

Comment:
This is ok, but usually `database.writeOptions` is used here. As those options might be out of date or in need of revision more than happy to let this one go.